test/jdk/ProblemList.txt

```diff
@@ -836,7 +836,6 @@ javax/swing/JMenuItem/4171437/bug4171437.java 8233641 macosx-all
 javax/swing/JMenu/4692443/bug4692443.java 8171998 macosx-all
 javax/swing/plaf/synth/7158712/bug7158712.java 8238720 windows-all
 javax/swing/plaf/basic/BasicComboPopup/JComboBoxPopupLocation/JComboBoxPopupLocation.java 8238720 windows-all
-javax/swing/plaf/basic/BasicComboPopup/7072653/bug7072653.java 8238720 windows-all
```

Comment:
Checked locally using different DPIs and on all our mach5 systems, all passed after the fix.